client/onboarding-prototype/form.tsx

```diff
@@ -74,6 +74,9 @@ export const OnboardingTextField: React.FC< OnboardingTextFieldProps > = ( {
 				if ( touched[ name ] ) validate( value );
 			} }
 			onBlur={ () => validate() }
+			onKeyDown={ ( event: React.KeyboardEvent< HTMLInputElement > ) => {
+				if ( event.key === 'Enter' ) validate( data[ name ] );
```

Comment:
_Nitpick:_ Doesn't make any difference, but we can call `validate()` here without the value, because it already takes the stored one if nothing is provided.

The same applies to the call in `OnboardingPhoneNumberField`.

Reply:
Ah, good to know! It's even more clever than I thought! 🤣 Fixed in 7ca7067ba.